src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -919,7 +921,8 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 				items.unshift({
 					label: formatLabel(executingCommand),
 					rawLabel: executingCommand,
-					description: cmdDetection.cwd
+					description: cmdDetection.cwd,
+					buttons: []
```

Comment:
`formatPinnedItems` should add a new array if `buttons` is `undefined`

Reply:
it does - these are leftover 👍🏼